templates/service-template.yml

```diff
@@ -167,6 +167,21 @@ parameters:
   description: Enable the Kafka TLS certificate
   value: "false"
 
+- name: RECONCILER_REPEAT_INTERVAL
+  displayName: Repeat Interval
+  description: The interval between cluster reconciliations.
+  value: "30s"
+
+- name: LEADER_ELECTION_RECONCILER_REPEAT_INTERVAL
+  displayName: Leader Election Repeat Interval
+  description: The interval between Leader Elections.
+  value: "15s"
+
+- name: LEADER_LEASE_EXPIRATION_TIME
+  displayName: Leaser Lease expiration time
+  description: This is the amount of time before a leader lease expires.
+  value: "1m"
```

Comment:
Have you tried to deploy the added parameters in OpenShift via code ready containers (crc) for example?

Reply:
Any updates on this?

Reply:
done